cypress/e2e/item/download/downloadItem.cy.ts

```diff
@@ -0,0 +1,27 @@
+import { DOWNLOAD_BUTTON_ID } from '@/config/selectors';
+import { ITEM_LAYOUT_MODES } from '@/enums';
+
+import { SHARED_ITEMS_PATH } from '../../../../src/config/paths';
+import { SHARED_ITEMS } from '../../../fixtures/sharedItems';
+
+describe('check if download button exists for all shared items despite the creator', () => {
```

Comment:
```suggestion
describe('Download Item', () => {
```

and if you want add another describe scope with you current title.